### ISSUE TITLE
Crashes on vm-reload with bigints

### ISSUE THREAD

**Issue by eernstg** [dart-lang/sdk#31660]
During gardening Dec 15, shortly after 9am CET, I noticed the following failures:
```
vm-linux-release-x64-reload-be:  
  none-vm-checked release_x64 corelib_2/bigint_test // Crash
  none-vm-checked release_x64 corelib_2/bigint_test // Crash
```
Here is the relevant log: [vm-linux-release-x64-reload-be](https://logs.chromium.org/v/?s=chromium%2Fbb%2Fclient.dart%2Fvm-linux-release-x64-reload-be%2F7991%2F%2B%2F%2A%2A%2Fstdout&s=chromium%2Fbb%2Fclient.dart%2Fvm-linux-release-x64-reload-be%2F7991%2F%2B%2F%2A%2A%2Fstderr).

The crashes seem to be associated with memory allocation:
```
Exhausted heap space, trying to allocate 1152921504606847008 bytes
```

**Comment by eernstg** [comment]
Status adjusted in 863609b7c97a7e0eb194d34d3eb6e394be38687b.

**Comment by crelier** [comment]
The implementation of Bigint has changed a lot since last December. It is not clear if this hot reload issue is still relevant.
@athomas, it would be very neat to have a tool probing the test logs for a given test on a particular configuration and return a list of historical status codes. This would make it much easier to clean up status files for tests that are marked as both passing and failing. If a test has not failed anymore in the last few dozen runs, it is probably safe to assume the underlying issue has been fixed and the status file entry should be removed.

**Comment by mkustermann** [comment]
This is really up to you. If you don't think it's worth investigating, please remove the `Crash` status file marker and close this issue.

**Comment by athomas** [comment]
@crelier Agree, the flakiness tool we have is based on the log files and, since this is marked as a flaky crash in the status files, wouldn't work. If you remove the Crash from the status files and it starts crashing again, this tool can find it:
dart tools/gardening/bin/compare_failures.dart vm-linux-release-x64-reload-be

Flakiness is definitely something we want to have better tooling for in the future.
